src/os/windows/crashlog_win.cpp

```diff
@@ -271,6 +271,7 @@ static const uint MAX_FRAMES     = 64;
 			/* Get symbol name and line info if possible. */
 			DWORD64 offset;
 			if (proc.pSymGetSymFromAddr64(hCur, frame.AddrPC.Offset, &offset, sym_info)) {
+				sym_info->Name[MAX_SYMBOL_LEN - 1] = '\0'; // SymGetSymFromAddr64 does not null-terminate truncated symbol names, ensure that symbol name is null-terminated
```

Comment:
As I read the doc, I don't think this is needed.

Reply:
After some testing, setting last element of allocated array to `\0` can be done only once, right after allocation.